### PR TITLE
NAS-119185 / 22.12.1 / Adding loading indicator to permissions editor

### PR DESCRIPTION
**Pull request by undsoft**
Also centers logo.

### CODE REVIEW

File: src/app/modules/layout/components/admin-layout/admin-layout.component.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
   line-height: 64px;
   max-width: var(--sidenav-width); /*15rem;*/
   overflow: hidden;
+  padding-right: 6px;
```
Comment (dszidi) [inline src/app/modules/layout/components/admin-layout/admin-layout.component.scss:39]: No need for padding as logo is already centered by way of the `justify-content: center` rule.

image

**Review by dszidi — COMMENTED** [review]
Other than the css issue, code looks good and loader works well.

image

File: src/app/modules/layout/components/admin-layout/admin-layout.component.scss (hunk 1 above)
Comment (undsoft) [reply src/app/modules/layout/components/admin-layout/admin-layout.component.scss:39]: It's centered programatically, not visually (easier to see on light themes).

Comment (dszidi) [reply src/app/modules/layout/components/admin-layout/admin-layout.component.scss:39]: I see what you mean. If we want to correct this, we should also consider that the logo appears elsewhere in the UI as well. I'll approve and leave it up to you whether we want to include this here or do something more global on another ticket

**Review by dszidi — APPROVED** [review]
LGTM

**Comment by undsoft** [comment]
Ok, I'll merge as is and then we can improve on it later.

**Pull request merged**